active/gridsyntax/GridSyntaxSpec2.md

```diff
@@ -0,0 +1,368 @@
+
+
+# Background
+_This adds a new short-hand Xaml syntax feature that makes it much easier to define a Grid in Xaml makrup. The proposal for this feature can be found [here.](https://github.com/microsoft/microsoft-ui-xaml/issues/673) The existing API documentation for Grid can be found [here.](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.grid)_
+
+_This file is an [updated copy](./GridSyntaxSpec.md), split off to make all of the lines commentable in PR. But this will merged back when complete._
```

Comment:
```suggestion
_This file is an [updated copy](./GridSyntaxSpec.md), split off to make all of the lines commentable in PR. But this will be merged back when complete._
```